packages/flutter/lib/src/material/switch.dart

```diff
@@ -161,7 +166,12 @@ class Switch extends StatelessWidget {
     super.key,
     required this.value,
     required this.onChanged,
+    @Deprecated(
+      'Use activeThumbColor instead. '
```

Comment:
Maybe we should also mention activeTrackColor? something like
```suggestion
      'Use activeThumbColor or activeTrackColor instead. '
```

Reply:
Updated, thank you.